src/com/palmergames/util/TimeMgmt.java

```diff
@@ -52,12 +52,12 @@ public static String formatCountdownTime(long l) {
 
 		String out = "";
 		if (l >= 3600) {
-			int h = (int) Math.floor(l / 3600);
+			int h = (int) Math.floor(l / 3600.0);
 			out = h + TownySettings.getLangString("msg_hours");
 			l -= h * 3600;
 		}
 		if (l >= 60) {
-			int m = (int) Math.floor(l / 60);
+			int m = (int) Math.floor(l / 60.0);

```

Comment:
I'm assuming this was an IDE thing?

Reply:
Math.floor() in this context isn't necessary at all. Doing something like `(int) (l / 60)` will automatically truncate the int which is basically flooring it. 
To prove this, here's a little simple, but seemingly infinite test code:
```java
for (long l = 0; l < Long.MAX_VALUE; ++l) {
    int f = (int) Math.floor(l / 3600.0);
    int t = (int) (l / 3600);
    if (f != t) {
        System.out.println("Comparison Inequality");
        break;
    }
}
```
hint: it doesn't output anything.